.github/workflows/unit-test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: make install
 
       - name: "Test raco fmt compliance"
-        run: raco fmt -i **/*.rkt && git diff --exit-code
+        run: (find . -type f -name '*.rkt' | xargs raco fmt -i) && git diff --exit-code
```

Comment:
Did you also fix `make hooks`? I think we should probably put the formatting code into a `make fmt` and then only call that everywhere.

Reply:
Oh no I did not

Reply:
Hilariously `make hooks` uses the better way, so that's convenient